bin/sr-run-ci-build.sh

```diff
@@ -24,7 +24,13 @@ if  [ "semaphore_docker" != $server_type ] && [ "local" != $server_type ] && [ "
   
   sudo apt-get install -y python3-dev libxml2-dev libxslt-dev python3-pip lcov wget git libssl-dev libffi-dev libyaml-dev
   sudo pip3 install --upgrade pip setuptools==51.1.1 gcovr
-  sudo pip3 install PyYAML==5.4.1 --ignore-installed
+  if [ "$ubuntu_version" != "jammy" ]
+  then
+    sudo pip3 install PyYAML==5.4.1 --ignore-installed
+  else
+    sudo pip3 install "PyYAML>=5.4.1" --ignore-installed
```

Comment:
Is there a reason that `PyYAML>=5.4.1` is in quotation marks while `PyYAML==5.4.1` is not?

Reply:
Yes, the `>` symbol would mean "redirect output" in bash if we don't double quote it. And it was making the script fail.
`==` doesn't need that.